src/FixedDataTable.js

```diff
@@ -639,7 +640,8 @@ var FixedDataTable = React.createClass({
   },
 
   _renderRows(/*number*/ offsetTop) /*object*/ {
-    var state = this.state;
+    let state = updateRowHeights(this.state, this.props);
+    state = updateVisibleRows(state);
```

Comment:
Collapsable rows need to re-calculate the row heights as part of the re-render. I'm not sure if hijacking from the scrollStateHelper like this is a good approach though.. thoughts?